scripts/processDiagnosticMessages.tsconfig.json

```diff
@@ -0,0 +1,56 @@
+{
+    "compilerOptions": {
+        "removeComments": false,
+        "outFile": "processDiagnosticMessages.js",
+        "target": "es5",
+        "declaration": false,
+        "lib": [
+            "es6",
+            "scripthost"
+        ]
+    },
+    "files": [
+        "../src/compiler/types.ts",
+        "../src/compiler/performance.ts",
+        "../src/compiler/core.ts",
+        "../src/compiler/sys.ts",
+        "../src/compiler/diagnosticInformationMap.generated.ts",
+        "../src/compiler/scanner.ts",
+        "../src/compiler/utilities.ts",
+        "../src/compiler/parser.ts",
+        "../src/compiler/binder.ts",
+        "../src/compiler/symbolWalker.ts",
+        "../src/compiler/moduleNameResolver.ts",
+        "../src/compiler/checker.ts",
+        "../src/compiler/factory.ts",
+        "../src/compiler/visitor.ts",
+        "../src/compiler/transformers/utilities.ts",
+        "../src/compiler/transformers/destructuring.ts",
+        "../src/compiler/transformers/ts.ts",
+        "../src/compiler/transformers/es2017.ts",
+        "../src/compiler/transformers/esnext.ts",
+        "../src/compiler/transformers/jsx.ts",
+        "../src/compiler/transformers/es2016.ts",
+        "../src/compiler/transformers/es2015.ts",
+        "../src/compiler/transformers/es5.ts",
+        "../src/compiler/transformers/generators.ts",
+        "../src/compiler/transformers/module/module.ts",
+        "../src/compiler/transformers/module/system.ts",
+        "../src/compiler/transformers/module/es2015.ts",
+        "../src/compiler/transformers/declarations/diagnostics.ts",
+        "../src/compiler/transformers/declarations.ts",
+        "../src/compiler/transformer.ts",
+        "../src/compiler/sourcemap.ts",
+        "../src/compiler/comments.ts",
+        "../src/compiler/emitter.ts",
+        "../src/compiler/watchUtilities.ts",
+        "../src/compiler/program.ts",
+        "../src/compiler/builderState.ts",
+        "../src/compiler/builder.ts",
+        "../src/compiler/resolutionCache.ts",
+        "../src/compiler/watch.ts",
+        "../src/compiler/commandLineParser.ts",
+
+        "processDiagnosticMessages.ts"

```

Comment:
Also need to change processDiagnosticMessages.ts to not include reference path in generated diagnostics file https://github.com/Microsoft/TypeScript/blob/master/scripts/processDiagnosticMessages.ts#L60